src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,72 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(","))) // Use Arrays.stream directly
+                .map(Integer::parseInt)
+                .filter(num -> num % 2 == 0)
+                .min(Integer::compare)
```

Comment:
```suggestion
                .min()
```
It’s  default logic.

Reply:
Нажаль після виправлення коду падає помилка, можливо залишити як було? 
'min(java.util.Comparator<? super java.lang.Integer>) in 'javautil.stream.Stream' cannot be applied to '(* :19

Reply:
you can fix this by using mapToInt instead of just map